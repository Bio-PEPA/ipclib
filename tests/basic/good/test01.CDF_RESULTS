0.1    0.126687
0.2    0.240304
0.3    0.341426
0.4    0.43085
0.5    0.509494
0.6    0.578328
0.7    0.638324
0.8    0.690425
0.9    0.735522
1    0.774441
1.1    0.807941
1.2    0.836708
1.3    0.861356
1.4    0.882434
1.5    0.900426
1.6    0.915758
1.7    0.928804
1.8    0.939888
1.9    0.949293
2    0.957264
2.1    0.964011
2.2    0.969716
2.3    0.974535
2.4    0.978603
2.5    0.982032
2.6    0.984921
2.7    0.987354
2.8    0.989399
2.9    0.991119
3    0.992564
3.1    0.993776
3.2    0.994794
3.3    0.995647
3.4    0.996362
3.5    0.99696
3.6    0.997462
3.7    0.997881
3.8    0.998232
3.9    0.998525
4    0.99877
4.1    0.998975
4.2    0.999146
4.3    0.999288
4.4    0.999407
4.5    0.999506
4.6    0.999589
4.7    0.999658
4.8    0.999716
4.9    0.999763
5    0.999803
5.1    0.999836
5.2    0.999864
5.3    0.999887
5.4    0.999906
5.5    0.999922
5.6    0.999935
5.7    0.999946
5.8    0.999955
5.9    0.999963
6    0.999969
6.1    0.999975
6.2    0.999979
6.3    0.999982
6.4    0.999985
6.5    0.999988
6.6    0.99999
6.7    0.999992
6.8    0.999993
6.9    0.999994
7    0.999995
7.1    0.999996
7.2    0.999997
7.3    0.999997
7.4    0.999998
7.5    0.999998
7.6    0.999998
7.7    0.999999
7.8    0.999999
7.9    0.999999
8    0.999999
8.1    0.999999
8.2    1
8.3    1
8.4    1
8.5    1
8.6    1
8.7    1
8.8    1
8.9    1
9    1
9.1    1
9.2    1
9.3    1
9.4    1
9.5    1
9.6    1
9.7    1
9.8    1
9.9    1
