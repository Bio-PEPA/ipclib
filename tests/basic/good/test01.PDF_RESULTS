0.1    1.20081
0.2    1.07251
0.3    0.951274
0.4    0.838747
0.5    0.735759
0.6    0.642548
0.7    0.558953
0.8    0.484552
0.9    0.418757
1    0.360894
1.1    0.310249
1.2    0.266106
1.3    0.227772
1.4    0.194592
1.5    0.165957
1.6    0.141309
1.7    0.120144
1.8    0.102009
1.9    0.0865003
2    0.0732626
2.1    0.0619817
2.2    0.0523833
2.3    0.0442281
2.4    0.0373082
2.5    0.0314438
2.6    0.0264795
2.7    0.0222818
2.8    0.0187358
2.9    0.0157433
3    0.01322
3.1    0.0110942
3.2    0.00930472
3.3    0.00779944
3.4    0.00653415
3.5    0.00547129
3.6    0.00457906
3.7    0.00383052
3.8    0.00320289
3.9    0.00267694
4    0.00223642
4.1    0.00186764
4.2    0.00155908
4.3    0.00130101
4.4    0.00108528
4.5    0.000905005
4.6    0.000754428
4.7    0.000628703
4.8    0.000523769
4.9    0.000436219
5    0.000363199
5.1    0.000302319
5.2    0.000251575
5.3    0.000209294
5.4    0.000174076
5.5    0.000144748
5.6    0.000120333
5.7    0.000100013
5.8    8.31059e-05
5.9    6.90419e-05
6    5.7346e-05
6.1    4.76217e-05
6.2    3.95385e-05
6.3    3.2821e-05
6.4    2.72396e-05
6.5    2.26033e-05
6.6    1.87528e-05
6.7    1.55555e-05
6.8    1.29012e-05
6.9    1.0698e-05
7    8.86964e-06
7.1    7.35262e-06
7.2    6.09414e-06
7.3    5.05031e-06
7.4    4.18466e-06
7.5    3.4669e-06
7.6    2.87185e-06
7.7    2.37861e-06
7.8    1.96983e-06
7.9    1.63109e-06
8    1.35043e-06
8.1    1.11792e-06
8.2    9.25335e-07
8.3    7.65836e-07
8.4    6.33756e-07
8.5    5.24396e-07
8.6    4.3386e-07
8.7    3.58915e-07
8.8    2.96885e-07
8.9    2.4555e-07
9    2.03071e-07
9.1    1.67924e-07
9.2    1.38846e-07
9.3    1.14793e-07
9.4    9.48977e-08
9.5    7.84435e-08
9.6    6.48365e-08
9.7    5.35852e-08
9.8    4.42826e-08
9.9    3.6592e-08
