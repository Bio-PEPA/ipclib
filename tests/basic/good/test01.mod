% |\      /|  ** The International PEPA compiler:
% | \    / |  ipc v0.99
% |  \  /  |  Produced on location in/between Edinburgh, 
% |   \/   |  Leeds, London, Ludlow, Lucca and Sophia-Antipolis
% |   <>   |  2002-8
% |   /\   |  Contributors:
% |  /  \  |      Jeremy Bradley, Allan Clark,
% | /    \ |      Stephen Gilmore and Fragkiskos Gounelas
% |/      \|  Queries or problems: a.d.clark@ed.ac.uk
% 
% This file was generated by ipc
% from the following pepa model

\model{




\statevector{
\type{ short }{ P1_2
 , P2_2
 , P1
 , P2
 , yyMasterProbeRunning
 , yyMasterProbeStopped
 , yyPassageProbeRunning
 , yyPassageProbeStopped }

}

\initial{
P1_2 = 1 ;
P2_2 = 0 ;
P1 = 1 ;
P2 = 0 ;
yyMasterProbeRunning = 0 ;
yyMasterProbeStopped = 1 ;
yyPassageProbeRunning = 0 ;
yyPassageProbeStopped = 1 ;

}

\transition{ trans_1_0 }
{
% a

\condition{ (P1_2 == 1) && (P1 == 1) && (yyMasterProbeStopped == 1) && (yyPassageProbeStopped == 1) }
\action{
next -> P1 = 0 ;
next -> yyMasterProbeStopped = 0 ;
next -> yyPassageProbeStopped = 0 ;
next -> P1_2 = 1 ;
next -> P2 = 1 ;
next -> yyMasterProbeRunning = 1 ;
next -> yyPassageProbeRunning = 1 ;

}

\rate{ 1 }
\priority{1}
}

\transition{ trans_1_1 }
{
% a

\condition{ (P1_2 == 1) && (P1 == 1) && (yyMasterProbeStopped == 1) && (yyPassageProbeStopped == 1) }
\action{
next -> P1_2 = 0 ;
next -> yyMasterProbeStopped = 0 ;
next -> yyPassageProbeStopped = 0 ;
next -> P2_2 = 1 ;
next -> P1 = 1 ;
next -> yyMasterProbeRunning = 1 ;
next -> yyPassageProbeRunning = 1 ;

}

\rate{ 1 }
\priority{1}
}

\transition{ trans_2_0 }
{
% a

\condition{ (P2_2 == 1) && (P1 == 1) && (yyMasterProbeRunning == 1) && (yyPassageProbeRunning == 1) }
\action{
next -> P1 = 0 ;
next -> yyPassageProbeRunning = 0 ;
next -> P2_2 = 1 ;
next -> P2 = 1 ;
next -> yyMasterProbeRunning = 1 ;
next -> yyPassageProbeStopped = 1 ;

}

\rate{ 1 }
\priority{1}
}

\transition{ trans_2_1 }
{
% b

\condition{ (P2_2 == 1) && (P1 == 1) && (yyMasterProbeRunning == 1) && (yyPassageProbeRunning == 1) }
\action{
next -> P2_2 = 0 ;
next -> yyMasterProbeRunning = 0 ;
next -> yyPassageProbeRunning = 0 ;
next -> P1_2 = 1 ;
next -> P1 = 1 ;
next -> yyMasterProbeStopped = 1 ;
next -> yyPassageProbeStopped = 1 ;

}

\rate{ 1 }
\priority{1}
}

\transition{ trans_5_0 }
{
% a

\condition{ (P2_2 == 1) && (P1 == 1) && (yyMasterProbeStopped == 1) && (yyPassageProbeStopped == 1) }
\action{
next -> P1 = 0 ;
next -> yyMasterProbeStopped = 0 ;
next -> yyPassageProbeStopped = 0 ;
next -> P2_2 = 1 ;
next -> P2 = 1 ;
next -> yyMasterProbeRunning = 1 ;
next -> yyPassageProbeRunning = 1 ;

}

\rate{ 1 }
\priority{1}
}

\transition{ trans_5_1 }
{
% b

\condition{ (P2_2 == 1) && (P1 == 1) && (yyMasterProbeStopped == 1) && (yyPassageProbeStopped == 1) }
\action{
next -> P2_2 = 0 ;
next -> P1_2 = 1 ;
next -> P1 = 1 ;
next -> yyMasterProbeStopped = 1 ;
next -> yyPassageProbeStopped = 1 ;

}

\rate{ 1 }
\priority{1}
}

\transition{ trans_3_0 }
{
% a

\condition{ (P1_2 == 1) && (P2 == 1) && (yyMasterProbeRunning == 1) && (yyPassageProbeRunning == 1) }
\action{
next -> P1_2 = 0 ;
next -> yyPassageProbeRunning = 0 ;
next -> P2_2 = 1 ;
next -> P2 = 1 ;
next -> yyMasterProbeRunning = 1 ;
next -> yyPassageProbeStopped = 1 ;

}

\rate{ 1 }
\priority{1}
}

\transition{ trans_3_1 }
{
% b

\condition{ (P1_2 == 1) && (P2 == 1) && (yyMasterProbeRunning == 1) && (yyPassageProbeRunning == 1) }
\action{
next -> P2 = 0 ;
next -> yyMasterProbeRunning = 0 ;
next -> yyPassageProbeRunning = 0 ;
next -> P1_2 = 1 ;
next -> P1 = 1 ;
next -> yyMasterProbeStopped = 1 ;
next -> yyPassageProbeStopped = 1 ;

}

\rate{ 1 }
\priority{1}
}

\transition{ trans_6_0 }
{
% a

\condition{ (P1_2 == 1) && (P2 == 1) && (yyMasterProbeStopped == 1) && (yyPassageProbeStopped == 1) }
\action{
next -> P1_2 = 0 ;
next -> yyMasterProbeStopped = 0 ;
next -> yyPassageProbeStopped = 0 ;
next -> P2_2 = 1 ;
next -> P2 = 1 ;
next -> yyMasterProbeRunning = 1 ;
next -> yyPassageProbeRunning = 1 ;

}

\rate{ 1 }
\priority{1}
}

\transition{ trans_6_1 }
{
% b

\condition{ (P1_2 == 1) && (P2 == 1) && (yyMasterProbeStopped == 1) && (yyPassageProbeStopped == 1) }
\action{
next -> P2 = 0 ;
next -> P1_2 = 1 ;
next -> P1 = 1 ;
next -> yyMasterProbeStopped = 1 ;
next -> yyPassageProbeStopped = 1 ;

}

\rate{ 1 }
\priority{1}
}

\transition{ trans_7_0 }
{
% b

\condition{ (P2_2 == 1) && (P2 == 1) && (yyMasterProbeRunning == 1) && (yyPassageProbeRunning == 1) }
\action{
next -> P2 = 0 ;
next -> yyMasterProbeRunning = 0 ;
next -> yyPassageProbeRunning = 0 ;
next -> P2_2 = 1 ;
next -> P1 = 1 ;
next -> yyMasterProbeStopped = 1 ;
next -> yyPassageProbeStopped = 1 ;

}

\rate{ 1 }
\priority{1}
}

\transition{ trans_7_1 }
{
% b

\condition{ (P2_2 == 1) && (P2 == 1) && (yyMasterProbeRunning == 1) && (yyPassageProbeRunning == 1) }
\action{
next -> P2_2 = 0 ;
next -> yyMasterProbeRunning = 0 ;
next -> yyPassageProbeRunning = 0 ;
next -> P1_2 = 1 ;
next -> P2 = 1 ;
next -> yyMasterProbeStopped = 1 ;
next -> yyPassageProbeStopped = 1 ;

}

\rate{ 1 }
\priority{1}
}

\transition{ trans_4_0 }
{
% b

\condition{ (P2_2 == 1) && (P2 == 1) && (yyMasterProbeRunning == 1) && (yyPassageProbeStopped == 1) }
\action{
next -> P2 = 0 ;
next -> yyMasterProbeRunning = 0 ;
next -> P2_2 = 1 ;
next -> P1 = 1 ;
next -> yyMasterProbeStopped = 1 ;
next -> yyPassageProbeStopped = 1 ;

}

\rate{ 1 }
\priority{1}
}

\transition{ trans_4_1 }
{
% b

\condition{ (P2_2 == 1) && (P2 == 1) && (yyMasterProbeRunning == 1) && (yyPassageProbeStopped == 1) }
\action{
next -> P2_2 = 0 ;
next -> yyMasterProbeRunning = 0 ;
next -> P1_2 = 1 ;
next -> P2 = 1 ;
next -> yyMasterProbeStopped = 1 ;
next -> yyPassageProbeStopped = 1 ;

}

\rate{ 1 }
\priority{1}
}



}

\solution{ 
\method{ automatic }
}

\passage{
\sourcecondition{ yyPassageProbeRunning }
\targetcondition{ yyMasterProbeStopped }
\t_start{ 0.5 }
\t_stop{ 10.0 }
\t_step{ 0.1 }
}

