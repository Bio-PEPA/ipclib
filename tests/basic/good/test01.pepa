// File: tests/good/test01.pepa
// Lexical analysis: testing comment syntax
r = 1.0;

P1 = (a, r).P2; // a Java-style comment
P2 = (b, r).P1; // a Java-style comment

P1 || P1      // A LaTeX-style comment 
