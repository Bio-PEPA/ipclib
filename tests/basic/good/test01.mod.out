Generator (0x8a55008): explore complete! (7 tangible states)
Initialising the functional analyser
SteadyStateAnalyser (0x9053058): starting Grassmann solve...
eliminating row 5...
solution complete.

status is 0
SteadyStateAnalyser (0x9053058): Grassmann complete (0.000000e+00 CPU seconds).
Initialising Uniformiser...
Uniformiser(0x97b0008) powering up...
Uniformiser(0x97b0008) performing passage time analysis...
Uniformiser(0x97b0008) there are 3 target states...
Uniformiser(0x97b0008) powered up!
Running uniformise();
Uniformiser(0x97b0008) ready (7 states)!
Uniformiser(0x97b0008) reading tangible states...
Uniformiser(0x97b0008) _q = 2
Uniformiser(0x97b0008) modified _q = 2.02
Uniformiser(0x97b0008) read complete ( 7 states, 14 transitions) 
Uniformiser(0x97b0008) preparing to assign values to t_max_hops and lhs...
Uniformiser(0x97b0008) done
Uniformiser(0x97b0008) max_hops is 93
Uniformiser(0x97b0008) setting length of sum_pi_target to 94
Uniformiser(0x97b0008) setting up pi_0...
Uniformiser(0x97b0008) done matrix multiplication (pi calculation)...
Uniformiser(0x97b0008) calculating answers for each t_point...
0.1    1.20081    PDF_DATA0
0.1    0.126687    CDF_DATA0
0.2    1.07251    PDF_DATA0
0.2    0.240304    CDF_DATA0
0.3    0.951274    PDF_DATA0
0.3    0.341426    CDF_DATA0
0.4    0.838747    PDF_DATA0
0.4    0.43085    CDF_DATA0
0.5    0.735759    PDF_DATA0
0.5    0.509494    CDF_DATA0
0.6    0.642548    PDF_DATA0
0.6    0.578328    CDF_DATA0
0.7    0.558953    PDF_DATA0
0.7    0.638324    CDF_DATA0
0.8    0.484552    PDF_DATA0
0.8    0.690425    CDF_DATA0
0.9    0.418757    PDF_DATA0
0.9    0.735522    CDF_DATA0
1    0.360894    PDF_DATA0
1    0.774441    CDF_DATA0
1.1    0.310249    PDF_DATA0
1.1    0.807941    CDF_DATA0
1.2    0.266106    PDF_DATA0
1.2    0.836708    CDF_DATA0
1.3    0.227772    PDF_DATA0
1.3    0.861356    CDF_DATA0
1.4    0.194592    PDF_DATA0
1.4    0.882434    CDF_DATA0
1.5    0.165957    PDF_DATA0
1.5    0.900426    CDF_DATA0
1.6    0.141309    PDF_DATA0
1.6    0.915758    CDF_DATA0
1.7    0.120144    PDF_DATA0
1.7    0.928804    CDF_DATA0
1.8    0.102009    PDF_DATA0
1.8    0.939888    CDF_DATA0
1.9    0.0865003    PDF_DATA0
1.9    0.949293    CDF_DATA0
2    0.0732626    PDF_DATA0
2    0.957264    CDF_DATA0
2.1    0.0619817    PDF_DATA0
2.1    0.964011    CDF_DATA0
2.2    0.0523833    PDF_DATA0
2.2    0.969716    CDF_DATA0
2.3    0.0442281    PDF_DATA0
2.3    0.974535    CDF_DATA0
2.4    0.0373082    PDF_DATA0
2.4    0.978603    CDF_DATA0
2.5    0.0314438    PDF_DATA0
2.5    0.982032    CDF_DATA0
2.6    0.0264795    PDF_DATA0
2.6    0.984921    CDF_DATA0
2.7    0.0222818    PDF_DATA0
2.7    0.987354    CDF_DATA0
2.8    0.0187358    PDF_DATA0
2.8    0.989399    CDF_DATA0
2.9    0.0157433    PDF_DATA0
2.9    0.991119    CDF_DATA0
3    0.01322    PDF_DATA0
3    0.992564    CDF_DATA0
3.1    0.0110942    PDF_DATA0
3.1    0.993776    CDF_DATA0
3.2    0.00930472    PDF_DATA0
3.2    0.994794    CDF_DATA0
3.3    0.00779944    PDF_DATA0
3.3    0.995647    CDF_DATA0
3.4    0.00653415    PDF_DATA0
3.4    0.996362    CDF_DATA0
3.5    0.00547129    PDF_DATA0
3.5    0.99696    CDF_DATA0
3.6    0.00457906    PDF_DATA0
3.6    0.997462    CDF_DATA0
3.7    0.00383052    PDF_DATA0
3.7    0.997881    CDF_DATA0
3.8    0.00320289    PDF_DATA0
3.8    0.998232    CDF_DATA0
3.9    0.00267694    PDF_DATA0
3.9    0.998525    CDF_DATA0
4    0.00223642    PDF_DATA0
4    0.99877    CDF_DATA0
4.1    0.00186764    PDF_DATA0
4.1    0.998975    CDF_DATA0
4.2    0.00155908    PDF_DATA0
4.2    0.999146    CDF_DATA0
4.3    0.00130101    PDF_DATA0
4.3    0.999288    CDF_DATA0
4.4    0.00108528    PDF_DATA0
4.4    0.999407    CDF_DATA0
4.5    0.000905005    PDF_DATA0
4.5    0.999506    CDF_DATA0
4.6    0.000754428    PDF_DATA0
4.6    0.999589    CDF_DATA0
4.7    0.000628703    PDF_DATA0
4.7    0.999658    CDF_DATA0
4.8    0.000523769    PDF_DATA0
4.8    0.999716    CDF_DATA0
4.9    0.000436219    PDF_DATA0
4.9    0.999763    CDF_DATA0
5    0.000363199    PDF_DATA0
5    0.999803    CDF_DATA0
5.1    0.000302319    PDF_DATA0
5.1    0.999836    CDF_DATA0
5.2    0.000251575    PDF_DATA0
5.2    0.999864    CDF_DATA0
5.3    0.000209294    PDF_DATA0
5.3    0.999887    CDF_DATA0
5.4    0.000174076    PDF_DATA0
5.4    0.999906    CDF_DATA0
5.5    0.000144748    PDF_DATA0
5.5    0.999922    CDF_DATA0
5.6    0.000120333    PDF_DATA0
5.6    0.999935    CDF_DATA0
5.7    0.000100013    PDF_DATA0
5.7    0.999946    CDF_DATA0
5.8    8.31059e-05    PDF_DATA0
5.8    0.999955    CDF_DATA0
5.9    6.90419e-05    PDF_DATA0
5.9    0.999963    CDF_DATA0
6    5.7346e-05    PDF_DATA0
6    0.999969    CDF_DATA0
6.1    4.76217e-05    PDF_DATA0
6.1    0.999975    CDF_DATA0
6.2    3.95385e-05    PDF_DATA0
6.2    0.999979    CDF_DATA0
6.3    3.2821e-05    PDF_DATA0
6.3    0.999982    CDF_DATA0
6.4    2.72396e-05    PDF_DATA0
6.4    0.999985    CDF_DATA0
6.5    2.26033e-05    PDF_DATA0
6.5    0.999988    CDF_DATA0
6.6    1.87528e-05    PDF_DATA0
6.6    0.99999    CDF_DATA0
6.7    1.55555e-05    PDF_DATA0
6.7    0.999992    CDF_DATA0
6.8    1.29012e-05    PDF_DATA0
6.8    0.999993    CDF_DATA0
6.9    1.0698e-05    PDF_DATA0
6.9    0.999994    CDF_DATA0
7    8.86964e-06    PDF_DATA0
7    0.999995    CDF_DATA0
7.1    7.35262e-06    PDF_DATA0
7.1    0.999996    CDF_DATA0
7.2    6.09414e-06    PDF_DATA0
7.2    0.999997    CDF_DATA0
7.3    5.05031e-06    PDF_DATA0
7.3    0.999997    CDF_DATA0
7.4    4.18466e-06    PDF_DATA0
7.4    0.999998    CDF_DATA0
7.5    3.4669e-06    PDF_DATA0
7.5    0.999998    CDF_DATA0
7.6    2.87185e-06    PDF_DATA0
7.6    0.999998    CDF_DATA0
7.7    2.37861e-06    PDF_DATA0
7.7    0.999999    CDF_DATA0
7.8    1.96983e-06    PDF_DATA0
7.8    0.999999    CDF_DATA0
7.9    1.63109e-06    PDF_DATA0
7.9    0.999999    CDF_DATA0
8    1.35043e-06    PDF_DATA0
8    0.999999    CDF_DATA0
8.1    1.11792e-06    PDF_DATA0
8.1    0.999999    CDF_DATA0
8.2    9.25335e-07    PDF_DATA0
8.2    1    CDF_DATA0
8.3    7.65836e-07    PDF_DATA0
8.3    1    CDF_DATA0
8.4    6.33756e-07    PDF_DATA0
8.4    1    CDF_DATA0
8.5    5.24396e-07    PDF_DATA0
8.5    1    CDF_DATA0
8.6    4.3386e-07    PDF_DATA0
8.6    1    CDF_DATA0
8.7    3.58915e-07    PDF_DATA0
8.7    1    CDF_DATA0
8.8    2.96885e-07    PDF_DATA0
8.8    1    CDF_DATA0
8.9    2.4555e-07    PDF_DATA0
8.9    1    CDF_DATA0
9    2.03071e-07    PDF_DATA0
9    1    CDF_DATA0
9.1    1.67924e-07    PDF_DATA0
9.1    1    CDF_DATA0
9.2    1.38846e-07    PDF_DATA0
9.2    1    CDF_DATA0
9.3    1.14793e-07    PDF_DATA0
9.3    1    CDF_DATA0
9.4    9.48977e-08    PDF_DATA0
9.4    1    CDF_DATA0
9.5    7.84435e-08    PDF_DATA0
9.5    1    CDF_DATA0
9.6    6.48365e-08    PDF_DATA0
9.6    1    CDF_DATA0
9.7    5.35852e-08    PDF_DATA0
9.7    1    CDF_DATA0
9.8    4.42826e-08    PDF_DATA0
9.8    1    CDF_DATA0
9.9    3.6592e-08    PDF_DATA0
9.9    1    CDF_DATA0
# elapsed matrix multiply time was -6.66107e-18 seconds for 93 iterations   DATA0
Uniformiser(0x97b0008) destructor...
Uniformiser(0x97b0008) powered down...

Execution complete.

